configureBuild.sh

```diff
@@ -159,6 +159,8 @@ setRepository() {
     suffix="corretto/corretto-${BUILD_CONFIG[OPENJDK_CORE_VERSION]:3}"
   elif [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_DRAGONWELL}" ]]; then
     suffix="alibaba/dragonwell${BUILD_CONFIG[OPENJDK_CORE_VERSION]/jdk/}"
+  elif [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_BISHENG}" ]]; then
+    suffix="feilongjiang/bishengjdk-11-mirror"
```

Comment:
If I read this correctly, this code is pulled from https://github.com/feilongjiang/bishengjdk-11-mirror which in turn comes from https://gitee.com/openeuler/bishengjdk-11/tree/risc-v. So basically we run third-party code on our stateful machines. Has anyone verified that the necessary security practices are in place? Do we have guidelines for that?

(To be clear, this is not a criticism of the PR or its author. I know we've pulled code from elsewhere in the past.)

Reply:
@aahlenst The security team will review the mirror.  There's some extra work to be done to also enable with build on Docker as std practice.

@feilongjiang Have you explored being able to build this in a Docker container?

Reply:
I only tested the native build on Ubuntu 18.04. Docker and cross-compile builds have not been tested yet.

Reply:
OK, can you see if you can get cross-compile and Docker builds going next?  We will eventually have a docker only build policy so might as well get ahead of that :-).

Reply:
Ok, I'll try next